Assets/Nickel/NickelBalanace/TheFirstPerson/Code/ExampleExtensions/Debug/DebugLine.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using TheFirstPerson;
+using UnityEngine;
+
+public class DebugLine : TFPExtension
```

Comment:
Why did you duplicate this folder from the others instead of just using theirs directly?

Reply:
If you want to keep this, you need to namespace them.